internal/config/config.go

```diff
@@ -134,18 +137,37 @@ func (c *Config) NewFabricClientForSDK(d *schema.ResourceData) *fabricv4.APIClie
 // Shim for Fabric tests.
 // Deprecated: when the acceptance package starts to contain API clients for testing/cleanup this will move with them
 func (c *Config) NewFabricClientForTesting() *fabricv4.APIClient {
-	client := c.newFabricClient()
+	client := c.newFabricClient(context.WithValue(context.Background(), ctxKey("fabricSDKv2"), "no_oauth_reload"))
```

Comment:
This is probably fine since it's only used in tests, but this could be an opportunity to define a client in the `acceptance` package for testing/cleanup instead.